### PR TITLE
install-deps.sh: install binutils 2.28 for xenial

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: install-deps.sh [hunk 1]
```diff
@@ -144,6 +140,19 @@ function ensure_decent_cmake_on_ubuntu {
...
 }
 
+function ensure_decent_binutils_on_ubuntu {
+    local old=$(ld --version | head -n1 | grep -Po ' \K[0-9].*')
+    if dpkg --compare-versions $old ge $new; then
+        return
+    fi
+    install_pkg_on_ubuntu \
+	binutils \
+	7fa393306ed8b93019d225548474c0540b8928f7 \
```
Comment (tchaikov) [inline install-deps.sh]: for those who are curious with this sha1, see https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;a=commit;h=7fa393306ed8b93019d225548474c0540b8928f7

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by tchaikov** [comment]
jenkins test make check arm64

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by adamemerson** [comment]
You might want to reference https://tracker.ceph.com/issues/42596

**Pull request merged**